modules/history/src/history.js

```diff
@@ -0,0 +1,195 @@
+const self = require("../index.js");
+const historyHelper = require("./history-helper.js");
+
+/**
+ * History module allows you to manage an history for your project.
+ * It use data-store module to stack entities on 'snapshot' method
+ * and restore them using 'go' method
+ * History module can be use to implement system like Undo/Redo, Ctrl+Z / Ctrl+Y
+ */
```

Comment:
Please be less verbose here. This description is already written in the doc :)

Reply:
I did this on purpose for people who don't read doc, some people only get info by reading the code